packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -55,6 +55,7 @@ export default class QuanticSearchInterface extends LightningElement {
     window.removeEventListener('hashchange', this.onHashChange);
   }
  
+  /** @param {import("coveo").SearchEngine} engine */
```

Comment:
can add this to initUrlManager as well tant qu'a y etre.

Reply:
Different PR for that :D I just needed some auto complete haha

Reply:
🤔 you're saying you're doing it in another PR already or you don't want to do it in this one for some other reason?

Reply:
@erocheleau  did you add this cause you have runtime error?

Reply:
No, I added it because I was trying something and I wanted autocomplete 😂
I can remove it, but I thought leaving it there would be useful in the future anyway?